docs/v2.4/reference/config.md

```diff
@@ -22,8 +23,8 @@ timeouts:
 logLevels:
   userDaemon: debug
 images:
-  registry: privateRepo
-  agentImage: ambassador-telepresence-agent:1.8.0
+  registry: privateRepo # This overrides the default docker.io/datawire repo
+  agentImage: ambassador-telepresence-agent:1.8.0 # This overrides the agent image to inject when intercepting
 grpc:
   maxReceiveSize: 10Mi
```

Comment:
should this example config maybe include a `cloud:` section? I have no strong feelings on it either way, cause i think the cloud config is specialized enough to warrant being skipped in the example, but wanna get your thoughts

Reply:
Doesn't hurt, so I'll add it!